components/mdns/include/mdns.h

```diff
@@ -830,6 +836,28 @@ esp_err_t mdns_unregister_netif(esp_netif_t *esp_netif);
  */
 esp_err_t mdns_netif_action(esp_netif_t *esp_netif, mdns_event_actions_t event_action);
 
+/**
+ * @brief   Browse mDns for a service `_service._proto`.
```

Comment:
Nitpick: mDns -> mDNS

Reply:
Oh, thanks for this correction, will fix it asap.